docs/faq.md

```diff
@@ -1,15 +1,48 @@
 #### What is the difference between refine.bio-processed and submitter-processed datasets?
 
+Samples and the datasets they comprise are designated as refine.bio-processed if we were able to obtain raw data in a suitable format for one of our processing pipelines. 
+If no suitable raw data is available for a sample on a supported platform, we obtain the _processed_ data available from the source repository and modify it to be more consistent with refine.bio-processed data; these samples are termed submitter-processed.
+See the **Source Data** section for more information.
+
 #### How do you process the data?
 
+We process samples run on supported microarray platforms with Single-Channel Array Normalization (SCAN) and transcriptomic samples run on supported sequencing platforms with Salmon and tximport.
+Please see the **Processing Information** section for more details.
+
 #### What type of data does refine.bio support?
 
+refine.bio currently supports gene expression data, specifically genome-scale microarray and RNA-seq data.
+
```

Comment:
Do we have list of supported platforms? Could be useful to link that here.

Reply:
We link to it in the source data section, but I can add the links here as well.

Reply:
I put the links in the question above it because that makes more sense to me, but I can be convinced otherwise.

Reply:
I think the follow-up question that would pop up in the user's brain would be either 'Which ones?' or 'Does it support XYZ thing I'm working with?' and that link would help provide a complete picture.